source/applications/dependency-injection.md

```diff
@@ -209,3 +215,28 @@ export default {
   initialize: initialize
 };
 ```
+
+### Getting an Application Instance from a Factory Instance
+
+[`Ember.getOwner`][4] will retrieve the application instance from an Ember
+Object that was instantiated from a registered factory. This means that
```

Comment:
I worked "owns" and "owner" into the added documentation. There is a description about the meaning of "owner" at the top of this documentation so I didn't want to repeat this too many times.